src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: var(--main-color);
+}
+
+.moyo-logo {
+  display: block;
+  width: 100%;
```

Comment:
width, object-fit and height do not do anything in this case. Can be removed

Reply:
Done.